gbfs-validator/versions/schemas/v3.0-RC/gbfs.json

```
@@ -0,0 +1,67 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/NABSA/gbfs/blob/v3.0-RC/gbfs.md#gbfsjson",
+  "description":
+    "Auto-discovery file that links to all of the other files published by the system.",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
+      "type": "integer",
+      "minimum": 1450155600
+    },
+    "ttl": {
+      "description":
+        "Number of seconds before the data in the feed will be updated again (0 if the data should always be refreshed).",
+      "type": "integer",
+      "minimum": 0
+    },
+    "version": {
+      "description":
+        "GBFS version number to which the feed conforms, according to the versioning framework (added in v1.1).",
+      "type": "string",
+      "const": "3.0-RC"
```

Let's add v3.0 to v3.0-RC schemas the same way as in https://github.com/MobilityData/gbfs-json-schema/pull/88. Thank you!

```suggestion
      "enum": [
        "3.0-RC",
        "3.0"
      ]
```

---

I've updated the files from https://github.com/MobilityData/gbfs-validator/pull/104#pullrequestreview-1491739949 so this is now included.

---

Thank you!